src/style.css

```diff
@@ -1,3 +1,80 @@
 body {
+  font-family: Roboto, Arial, sans-serif;
   margin: 0;
+  padding: 0;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  background-color: #fff;
+  padding: 0 50px;
+  max-width: 1200px;
+}
+
+.go-home {
+  margin: 10px 0;
+  height: 40px;
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+}
+
+.nav {
+  display: flex;
+}
+
+.nav__list {
+  display: flex;
+  align-items: center;
+  list-style: none;
+  text-transform: uppercase;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100%;
+}
+
+.nav__item:not(:first-child) {
+  margin-left: 20px;
+}
+
+.nav__link {
+  display: flex;
+  align-items: center;
+  text-decoration: none;
+  line-height: normal;
+  font-size: 12px;
+  font-weight: 500;
+  color: #000;
+  font-style: normal;
+  height: 100%;
```

Comment:
why you need to set this `height`?

Reply:
Because links should have clickable area above and below the text. If I remove this height, clickable area disappears.

Reply:
> why you need to set this `height`?

Because links should have clickable area above and below the text. If I remove this height, clickable area disappears.